.github/workflows/release_benchmark_fast.yaml

```diff
@@ -0,0 +1,50 @@
+name: Build Benchmark Fixtures
+
+on:
+  push:
+    branches:
+      # List specific branches for now.
+      - "forks/osaka"
+      - "forks/amsterdam"
```

Comment:
I think we should only generate for the default fork, especially as we only generate benchmark fixtures for Prague. I guess this will be Amsterdam very soon :)